utils/test/jest.config.js

```diff
@@ -9,6 +9,7 @@ const path = require('path');
 const { defaults } = require('jest-config');
 
 module.exports = {
+  testEnvironment: 'jsdom',
```

Comment:
Node projects that used Jest sometimes unknowingly used the default `jsdom` environment, which is expensive. In v27, Jest now defaults to `node`, so this project needs to explicitly ask for the `jsdom` environment.

See: https://jestjs.io/blog/2021/05/25/jest-27